decimal_test.go

```diff
@@ -755,8 +756,6 @@ func TestJSONEncoding(t *testing.T) {
 		"1234567890",
 		"298472983472983471903246121093472394872319615612417471234712061",
 		"0.0",
-		"NaN",
```

Comment:
What are the test failures with these?

Reply:
`Nan` and `Inf` aren't valid JSON numbers.
http://www.json.org/
http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf